hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -54,19 +54,22 @@
         "item_type": "file.photoshop",
     },
     "Rendered Image": {
-        "extensions": ["dpx", "exr"],
+        "extensions": ["dpx", "exr", "cin"],
         "icon": "image_sequence.png",
         "item_type": "file.image",
     },
     "Texture Image": {
-        "extensions": ["tiff", "tx", "tga", "dds", "rat"],
+        "extensions": ["tiff", "tx", "tga", "dds", "rat", "cin"],
```

Comment:
FYI, the way this dictionary is used, it searches each extension list to identify a common file type. So when it encounters `cin` in one list, it will always use that type. In other words, the extension lists should be mutually exclusive (at least in the way that the default implementation works).